css/emoji-mart.css

```diff
@@ -31,6 +31,41 @@
   border-bottom-left-radius: 5px;
   border-bottom-right-radius: 5px;
 }
+.emoji-mart-bar.emoji-mart-skin-tones {
+  position: relative;
+  height: 49px;
+}
+
+.emoji-mart-bar.emoji-mart-skin-tones .emoji-mart-preview-skins {
+  display: flex;
+  height: 48px;
+  align-items: center;
+  position: unset;
+  transform: none;
+  margin-left: 8px;
+}
+
+.emoji-mart-skin-tones-add-emoji {
+  position: absolute;
+  right: 8px;
+  top: 50%;
+  transform: translateY(-50%);
+}
+
+.emoji-mart-skin-tones-add-emoji-button {
```

Comment:
do we still need these or are they just for the new story? i'd be okay with just an unstyled button in the story if it meant we could drop this extra css.

Reply:
yeah, it's just for the story, so I'll go ahead and remove it 👍